tests/update_reference_data.sh

```diff
@@ -194,6 +194,7 @@ tests[bo_diffusion]="flow diffusion BO_DIFFUSE_CASE1"
 tests[fpr_nonhc]="flow water-1ph WATER2F"
 tests[actionx_wpimult]="flow actionx ACTIONX_WPIMULT"
 tests[wvfpexp_02]="flow wvfpexp WVFPEXP-02"
+tests[spe1case2_krnum=]"flow spe1 SPE1CASE2_KRNUM"
```

Comment:
Oops.  It seems you've accidentally swapped the assignment and the right bracket (`]`) here.

Reply:
this is getting embarrassing :-). Now this should be good. Again very sorry